aqua/extraAquaSettings.php

```diff
@@ -33,10 +33,9 @@ function ModifyDefaultLogin(&$personal_urls, &$wgTitle) {
 
 $wgHooks['PersonalUrls'][] = 'ModifyDefaultLogin';
 
-# Disable reading by anonymous users
-# WARNING: If you uncomment this line, remote verification via API will be
-# denied access.
-# $wgGroupPermissions['*']['read'] = false;
+# Disable reading by anonymous users using `./pkc setup --private -w WALLET_ADDRESS`
+# WARNING: remote verification via API will be denied access.
```

Comment:
This warning doesn't explain whether the true/false state denies the API access. Needs to start with "If false, ...".